include/stomp_moveit/cost_functions.hpp

```diff
@@ -1,84 +1,79 @@
 #pragma once
 
 #include <Eigen/Geometry>
+
 #include <moveit/planning_scene/planning_scene.h>
+#include <moveit/kinematic_constraints/kinematic_constraint.h>
+
 #include <stomp_moveit/stomp_moveit_task.hpp>
 #include <stomp_moveit/conversion_functions.hpp>
 
 namespace stomp_moveit
 {
-namespace costs
+namespace
```

Comment:
Would it make sense to split this into two header files (one for each namespace?) for cleaner separation?

Reply:
Maybe you're right. It could even be part of the public API, so that people can create their own cost functions from binary checks

Reply:
... now that I think about this, we should leave it in the same namespace since it's just another cost function after all